modules/db2/src/main/java/org/testcontainers/containers/Db2Container.java

```diff
@@ -69,10 +72,8 @@ protected void configure() {
         addEnv("DB2INST1_PASSWORD", password);
 
         // These settings help the DB2 container start faster
-        if (!getEnvMap().containsKey("AUTOCONFIG"))
-            addEnv("AUTOCONFIG", "false");
-        if (!getEnvMap().containsKey("ARCHIVE_LOGS"))
-            addEnv("ARCHIVE_LOGS", "false");
+        if (!getEnvMap().containsKey("AUTOCONFIG")) addEnv("AUTOCONFIG", "false");
```

Comment:
hmmm, interesting. I was under a strong impression that our codestyle disallows single statement `if`s? Could you please check why it didn't change to `if (...) {}`?

Reply:
it is because of [this](https://github.com/testcontainers/testcontainers-java/blob/master/config/checkstyle/checkstyle.xml#L19)

Reply:
ha, interesting! Can we disable it for all literals but `LAMBDA`?